configuration/chirpstack/region_eu868.toml

```diff
@@ -0,0 +1,132 @@
+# This file contains an example EU868 configuration.
+[[regions]]
```

Comment:
I'm not super sharp in toml files, but theres a mix of single brackets and double brackets here?

Reply:
Me neither. This is taken from the chirpstack documentation (where I made minor changes), so I think it should be this way..